platform/app/src/routes/WorkList/WorkList.tsx

```diff
@@ -382,16 +386,30 @@ function WorkList({
                     // to={`${mode.routeName}/dicomweb?StudyInstanceUIDs=${studyInstanceUid}`}
                   >
                     {/* TODO revisit the completely rounded style of buttons used for launching a mode from the worklist later - for now use LegacyButton*/}
-                    <LegacyButton
-                      rounded="full"
-                      variant={isValidMode ? 'contained' : 'disabled'}
+                    <Button
+                      type={ButtonEnums.type.primary}
+                      size={ButtonEnums.size.medium}
                       disabled={!isValidMode}
-                      endIcon={<Icon name="launch-arrow" />} // launch-arrow | launch-info
+                      tooltip={
+                        !isValidMode ? (
+                          <div className="font-inter flex w-[206px] whitespace-normal text-left text-xs font-normal text-white	">
+                            This study does not contain modalities supported by {mode.displayName}{' '}
```

Comment:
I think the mode itself should provide this text, maybe isValid on the mode should return an object a boolean and a description.

Reply:
For intstance for tmtv, we can say it is activated on studies with both ct and pt

For SEG it is a little bit more 

- on SM -> no support for seg on SM yet
- on DX, MG -> no support for non-volumetric segmentation yet

etc etc

Check wordings with Dan

Reply:
Done, wording is my own though, I didn't check with Dan on this yet.